packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -388,7 +388,12 @@ export class SummaryGenerator {
 				return fail("disconnect");
 			}
 			if (waitBroadcastResult.result !== "done") {
-				return fail("summaryOpWaitTimeout");
+				// If summary op was not received, it could just be a transient error. So,
+				// fail with a retriable error which may result in another attempt.
```

Comment:
summary op might not have been received because of some transient error, so here we fail with a retriable error to re-attempt another summary.

Reply:
Re-worded it. The re-attempt may not happen depending on how many attempts have happened yet. So, I kept that part of the comment.